recipes-app/node-red-node-serialport/node-red-node-serialport_1.0.1.bb

```diff
@@ -14,6 +14,7 @@ DESCRIPTION = "Node-RED nodes to talk to serial ports"
 
 NPM_LOCAL_INSTALL_DIR = "/root/.node-red"
 
-do_install_append() {
-    rm -rf $(find ${D} -name prebuilds -type d)
+do_prepare_build_append() {
+    sed -i '/override_dh_install:/a\\trm -r ${PP}/image/root/.node-red/node_modules/node-red-node-serialport/node_modules/@serialport/bindings-cpp/prebuilds' \
```

Comment:
This is based on the assumption that `override_dh_install` is present. Better check in advance as this otherwise might fail silently in a future version.

Reply:
Both class and recipe are in the same layer. And if this rule is not patched properly, the build will fail anyway.